samples/system-test/spanner.test.js

```diff
@@ -259,21 +259,41 @@ describe('Autogenerated Admin Clients', () => {
       await sample_instance.delete();
     });
 
-    // create_instance
-    it('should create an example instance', async () => {
-      const output = execSync(
+    // create_and_update_instance
+    it('should create and update an example instance with spanner editions', async () => {
+      const createInstanceOutput = execSync(
         `${instanceCmd} createInstance "${SAMPLE_INSTANCE_ID}" ${PROJECT_ID}`
       );
       assert.match(
-        output,
+        createInstanceOutput,
         new RegExp(
           `Waiting for operation on ${SAMPLE_INSTANCE_ID} to complete...`
         )
       );
       assert.match(
-        output,
+        createInstanceOutput,
         new RegExp(`Created instance ${SAMPLE_INSTANCE_ID}.`)
       );
+
+      const updateInstanceOutput = execSync(
+        `${instanceCmd} updateInstance "${SAMPLE_INSTANCE_ID}" ${PROJECT_ID}`
```

Comment:
Instead of calling the same instanceCmd with "updateInstance" , you can directly invoke the "instance-update.js" file

Reply:
done